server/client/ruby/candlepin_api.rb

```diff
@@ -79,14 +79,16 @@ def register(name, type=:system, uuid=nil, facts={}, username=nil,
               environment=nil, capabilities=[], hypervisor_id=nil,
               content_tags=[], created_date=nil, last_checkin_date=nil,
               annotations=nil, recipient_owner_key=nil, user_agent=nil,
-              entitlement_count=0, id_cert=nil, offering=nil, usage=nil)
+              entitlement_count=0, id_cert=nil, role=nil, usage=nil,
```

Comment:
`offering` is a dead letter now I guess?

Reply:
Ya, it appears that offering has now become 'role'.